### ISSUE TITLE
Develop example of horizontal menubar

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#154]
As discussed in issue#120, due to the complex nature of arrow key requirements for menubars that open submenus that open submenus ... it is not easy for users to understand the effects of orientation on navigation without an example.

One of the examples being reviewed in issue #145 could be reworked to fill this need.

**Comment by a11ydoer** [comment]
@mcking65 @jongund @jnurthen 

It would be hard to meet December public working draft deadline(Dec 5) with vertical menu bar since we are still working on horizontal menu bar left and right arrow issues plus Mac vs PC keyboard model. As I mentioned in the meeting, can the vertical menubar be a separate example?

**Comment by mcking65** [comment]
@a11ydoer 

Yes, this is for a separate example page to be included in the Spring CR publication. We could push it out to release 2 if it is not containable for March/April as this is a much less common need. I edited the description to clarify.

**Comment by a11ydoer** [comment]
@mcking65 
Do you think we still need this example?
